src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -929,9 +935,6 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 			const newIndexOfEditor = this.getIndexOfEditor(editor);
 			if (newIndexOfEditor !== oldIndexOfEditor) {
 				this.titleAreaControl.moveEditor(editor, oldIndexOfEditor, newIndexOfEditor);
-
-				// Event
-				this._onDidGroupChange.fire({ kind: GroupChangeKind.EDITOR_MOVE, editor });
```

Comment:
@bpasero Let me know if you're ok with this. We now just listen to all the model changes and forward those as listening to both the model and firing these was causing tests to fail.

Reply:
If I understand this change correctly, we no longer manually emit the `EDITOR_MOVE` from here but have a listener on the group model to re-emit when it happens?

I think that is probably fine, there seem to be only 2 listeners: your main thread tabs and the "Open Editors" view.